app/helpers/application_helper.rb

```diff
@@ -24,6 +24,10 @@ def lamports_to_sol(lamports)
     lamports / 1_000_000_000.to_f
   end
 
+  def lamports_to_sol_secure(lamports)
```

Comment:
Do we need this method? It seems really redundant.
You can just convert variable to_f, or it can even be done inside existing lamports_to_sol method